pycbc/inference/io/gwosc.py

```diff
@@ -0,0 +1,239 @@
+# Copyright (C) 2019 Collin Capano
```

Comment:
Does this need a write posterior method?

Reply:
Yeah, I suppose it does.